packages/lesswrong/server/resolvers/userResolvers.ts

```diff
@@ -301,3 +301,138 @@ createPaginatedResolver({
     return await context.repos.users.getSubscriptionFeedSuggestedUsers(currentUser._id, limit);
   }
 });
+
+addGraphQLSchema(`
+  type NetKarmaChangesForAuthorsOverPeriod {
+    userId: String,
+    netKarma: Int
+  }
+`)
+
+defineQuery({
+  name: "NetKarmaChangesForAuthorsOverPeriod",
+  argTypes: "(days: Int!, limit: Int!)",
+  resultType: "[NetKarmaChangesForAuthorsOverPeriod!]!",
+  fn: async (
+    _root: void,
+    {days, limit}: {days: number, limit: number},
+    context: ResolverContext,
+  ) => {
+    return await context.repos.votes.getNetKarmaChangesForAuthorsOverPeriod(days, limit);
+  }
+});
+
+type AirtableLeaderboardResultType = {
+  name: string;
+  leaderboardAmount?: number;
+};
+
+
+async function fetchAirtableRecords(): Promise<AirtableLeaderboardResultType[]> {
+  const baseId = "appUepxJdxacpehZz";
+  const tableName = "Donors";
+  const viewName = "LeaderBoard";
+
+  const apiKey = airtableApiKeySetting.get();
+  if (!apiKey) {
+    throw new Error("Can't fetch Airtable records without an API key");
+  }
+
+  const url = `https://api.airtable.com/v0/${baseId}/${encodeURIComponent(tableName)}?view=${encodeURIComponent(viewName)}`;
+  const response = await fetch(url, {
+    headers: {
+      Authorization: `Bearer ${apiKey}`,
+    },
+  });
+
+  if (!response.ok) {
+    throw new Error(`Airtable API responded with status ${response.status} - ${response.statusText}`);
+  }
+
+  // The data returned by Airtable has a "records" array.
+  // Each "record" has a "fields" object containing the columns we need.
+  const data = await response.json();
+  const records = data.records || [];
+
+  const unpackArray = (value: any) => Array.isArray(value) ? value[0] : value;
```

Comment:
prefer to define this outside the query... also `value` should probably be `AnyBecauseIsInput`.